app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -305,9 +305,18 @@ protected void onDestroy() {
         super.onDestroy();
         // unregister our receiver
         this.unregisterReceiver(this.mReceiver);
+        KissApplication.getCameraHandler().releaseCamera();
```

Comment:
in most case, the `getCameraHandler()` will be called here for the first time. So this means we'll have to do `Camera.open()` and release it immediately.

Maybe we should first check if this has already been loaded?